ansible/www-standalone/tools/promote/_resha.sh

```diff
@@ -25,3 +50,6 @@ nodejs-dist-indexer --dist $dstdir --indexjson ${dstdir}/index.json  --indextab
 find "${dstdir}/${version}" -type f -exec chmod 644 '{}' \;
 find "${dstdir}/${version}" -type d -exec chmod 755 '{}' \;
 
+relativedir=${dstdir/$dist_rootdir/"$site/"}
+aws s3 cp ${dstdir}/index.json $staging_bucket/$relativedir/index.json --endpoint-url=$cloudflare_endpoint --profile $cloudflare_profile
+aws s3 cp ${dstdir}/index.tab $staging_bucket/$relativedir/index.tab --endpoint-url=$cloudflare_endpoint --profile $cloudflare_profile
```

Comment:
Shouldn't we also copy the two shasum files?

Reply:
The shasum files are already put into the dist-prod bucket by the release script, https://github.com/nodejs/node/blob/a923fed8743cf787add2a22def0fdbb88d736a42/tools/release.sh#L20C1-L20C18

Reply:
_resha.sh is not only called by releasers. Automation for nightly, test, rc, builds also calls it.

Reply:
Opened nodejs/node#53017 to upload the shasums to the staging bucket then

Reply:
Maybe? I'm not entirely sure about the order in which things happen during a release. 
I noticed yesterday when I released v22.2.0 is that `SHASUMS256.txt` wasn't uploaded to R2 (only the `.asc` and `.sig` files were).

Reply:
In that case maybe we just copy over the SHASUM256.txt, @MoLow would you have any input on this?

Reply:
From what it looks like in the release guide doc (https://github.com/nodejs/node/blob/main/doc/contributing/releases.md#15-promote-and-sign-the-release-builds) the CI run is done first. This uploads the binaries, docs, and `SHASUM256.txt` to the `dist-staging` bucket.

When promoting, the `SHASUM256.txt.asc` and `SHASUM256.txt.sig` files are created and uploaded. Imo the release script is fine as it is with uploading directly to the `dist-prod` bucket, skipping `dist-staging`. The .asc and .sig files are still uploaded to the DO server as well ofc

Reply:
Changed it so that it will copy the SHASUM256.txt into the staging bucket now. The later call that syncs the version in the staging bucket -> dist bucket recursively copies the entire directory, so no change should be needed there.

Reply:
CI doesn't upload Shasums. For releases it happens when the release script is executed

Reply:
`SHASUM256.txt` is generated on the www machine by the promotion script (specifically [`_resha.sh`](https://github.com/nodejs/build/blob/main/ansible/www-standalone/tools/promote/_resha.sh), which also regenerates `index.json` and `index.tab`). For releases promotion is manually triggered by the releaser. For nightly builds there's a 30 min cron job that runs [`promote_nightly.sh`](https://github.com/nodejs/build/blob/main/ansible/www-standalone/tools/promote/promote_nightly.sh):
https://github.com/nodejs/build/blob/3d2cdcca83b16b35aa9ec9a4315bb71687141d83/ansible/www-standalone/tasks/site-setup.yaml#L63

The signed artifacts (`SHASUM256.txt.asc` and `SHASUM256.txt.sig`) are only generated for releases as part of the [release script](https://github.com/nodejs/node/blob/main/tools/release.sh). These are generated on the releaser's machine and uploaded as part of that script.

Reply:
@flakey5 I assume you have updated the code based on the comments above?

Reply:
From my understanding yes